content/posts/soumyadeep-dubey/the-5-best-exercises-to-build-muscle/index.md

```diff
@@ -0,0 +1,53 @@
+---
+template: "post"
+slug: "the-5-best-exercises-to-build-muscles"
+
+title: "The 5 Best Exerciese-to-build-muscles"
+author: "Soumyadeep Dubey"
+
+cover: "./banner.jpg"
+images:
```

Comment:
all images used in the blog has to have a meta, you only defined the `banner` image. See [this](https://github.com/thedarkale/website4/blob/main/content/posts/aniruddha-pandey/1001-movies-you-must-see-before-you-die/index.md) for example.